scripts/index.js

```diff
@@ -0,0 +1,97 @@
+//Temperature values elements that will change when clicked
+const tempContainer = document.getElementById('temperature-value');
+const increaseButton = document.getElementById('increaseTemp');
+const decreaseButton = document.getElementById('decreaseTemp');
+const gardenWeather = document.querySelector('.garden-weather');
+const skyWeather = document.querySelector('.sky-weather');
+const skyType = document.getElementById('sky-type');
+const resetButton = document.getElementById('resetButton');
+const cityName = document.querySelector('.city');
+let cityInput = document.querySelector('.cityInput')
+
+const landscapeImage= {
+  summer:	"🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂",
+  spring: "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷",
+  fall:	"🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃",
+  winter: "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲",
+}
+const skyImage = {
+  sunny: "☀️☁️ ☀️☁️ ☀️☁️ ☀️ ☁️ ☀️ ☁️☀️",
+  cloudy:	"☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️",
+  rainy: "🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧",
+  snowy: 	"🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨",
+}
+
+const state = {
+  tempCount: 76,
+  currentLandscapeImage: landscapeImage.spring
+};
+//increases and decrease tempereature with one click 
+const addTemperature = () => {
+  state.tempCount += 1;
+  tempContainer.textContent = `${state.tempCount}`;
+  changeTempTextColor();
+}
+const decreaseTemperature = () => {
+  state.tempCount -= 1;
+  tempContainer.textContent = `${state.tempCount}`;
+  changeTempTextColor();
+}
+
+const changeTempTextColor = () => {
+  if(state.tempCount >= 80) {
```

Comment:
A chained conditional `if\else if\else if` would make this code a bit more efficient. Also, minor note, if orangize it as `temp <= 49 / else if temp <= 59 ...` we do not need the compound conditional.